main.js

```diff
@@ -271,6 +287,50 @@
         exports._stateManager = _stateManager;
         exports._assetManagers = _assetManagers;
 
+        if (!!_config["meta-data-test-menu-enabled"]) {
+            
+            var onMetaDataTestMenu = function (event) {
+                if (event.generatorMenuChanged.name === "CREMA-META-DATA-TEST") {
+                    generator.getDocumentInfo(null, null).then(function (gDoc) {
+                        var doc = JSON.parse(JSON.stringify(gDoc)),
+                            layerId;
+
+                        generator.setDocumentSettingsForPlugin({
+                            metaEnabled: true,
+                            scaleSettings: [{
+                                scale: 1
+                            }, {
+                                folder: "scaled-at-25/",
+                                scale: 0.25
+                            }, {
+                                folder: "scaled-at-200/",
+                                suffix: "@2x",
+                                scale: 2
+                            }]
+                        }, META_PLUGIN_ID);
+
+                        //apply some settings to the currently selected layer
+                        if (doc.selection.length > 0) {
+                            layerId = findLayerIdForIndex (doc.layers, doc.selection[0]);
+                            generator.setLayerSettingsForPlugin({
+                                assetSettings: [{
+                                    name: "30x? layer.png-32",
```

Comment:
Will the meta-data need the "name"? Doesn't that just come from the layer or do we still need to add that? Maybe safer to add that in the component manager so it keeps it in sync.